etc/profile-m-z/nextcloud-desktop.profile

```diff
@@ -0,0 +1,48 @@
+# Firejail profile for nextcloud-desktop
+# Description: Nextcloud desktop client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include nextcloud-desktop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${DOCUMENTS}
+noblacklist ${HOME}/Nextcloud
```

Comment:
I'm not sure if we want to add a blacklist for that? For what is it? If this is the place where all file are synced to, almost every program should have access to it.

Reply:
In QOwnNotes.profile, we have the same.
That's the default for Nextcloud.
But I really understand you point

Reply:
QOwnNotes has only `(no)blacklist ${HOME}/Nextcloud/Notes`. But if there are audio/video/document/... files we would need to add a noblacklist to all that viewers/editors.

Reply:
Yes, I think it could be something to think about. At least for images. Maybe only opening `whitelist ${DOWNLOADS}`.
What do you think?
BTW, I tried just now and I can load save images in QOwnNotes.
I cannot see buttons to load video or audio. So, only that whitelist should be enough

Reply:
> I'm not sure if we want to add a blacklist for that? For what is it? If this is the place where all file are synced to, almost every program should have access to it.

now we don't need those lines anymore

```
#noblacklist ${HOME}/Nextcloud
#noblacklist ${HOME}/.config/Nextcloud
#noblacklist ${HOME}/.local/share/Nextcloud
```

I'm going to delete them from the next commit

Reply:
We need `noblacklist ${HOME}/.config/Nextcloud` and `noblacklist ${HOME}/.local/share/Nextcloud` after adding `blacklist`s for them in disable-programs.inc.